docs/4_4_1_can.adoc

```diff
@@ -270,9 +268,10 @@ For this operation applies: `Total Length = 9`.
 |The ID of the confirmed CAN message.
 
 h|Behavior
-3+|The specified operation shall be produced by the Bus Simulation and consumed by a Network FMU.
-If `org.fmi-standard.fmi-ls-bus.DirectConfirmation` (see <<direct-confirmation-parameter>>) is set to `true` the confirmation of the transmitted network data shall be directly done by the sender itself.
-Otherwise the confirmation of the transmitted network data shall be done by the Bus Simulation.
+3+|The specified operation shall be produced by the Bus Simulation and consumed by Network FMUs.
+If the structural parameter `org.fmi-standard.fmi-ls-bus.WaitForBusNotification` (see <<bus-notification-parameter>>) is set to `false`, the Network FMU does not rely on receiving Confirm operations.
+In this case, Bus Simulations should not send Confirm operations to the Network FMU.
+If all Network FMUs, except the <<TransmitOpCodeCanLowCut, Transmit operation>> initiating Network FMU, communicated the <<StatusOpCodeCanLowCut, status>> `BUS_OFF` the Bus Simulation shall not provide a confirmation.
```

Comment:
Grammer? I have trouble understanding the sentence in line 274

Reply:
You are right, we will fix this